src/StringUtility.h

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2021, Group5 Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file. See the AUTHORS file for names of contributors.
+
+#pragma once
+#include <vector>
+#include <string>
+
+namespace StringUtility {
+	std::vector<std::string> split(const std::string& str, const std::string& del);
+	std::string SHA1code(const std::string& str);
```

Comment:
SHA1 is a hash, so it doesn't code. Consider using just "SHA1".

Reply:
done